src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_COMMENT = new Prefix("c/");
+    public static final Prefix PREFIX_TUTORIAL = new Prefix("t/");
```

Comment:
Is it better to differentiate this with the `t` for tags ?

Reply:
Currently there is no need to differentiate since Attendance isn't part of AddCommand and EditCommand. However, in the future when I improve the Attendance feature, I will make the necessary changes